api/serializers/student/internal.py

```diff
@@ -9,6 +9,10 @@
 
 
 class StudentWriteSerializer(serializers.ModelSerializer[Student]):
+    teaching_languages_and_levels = serializers.PrimaryKeyRelatedField(
+        queryset=LanguageAndLevel.objects.all(), required=False, allow_empty=True, many=True
```

Comment:
That's a tricky one. On the one hand we do, because there are students that get no level, but on the other hand we should make sure language and level is not omitted by mistake. I guess that, ideally, we should make it optional but throw an exception if level is empty but `project_status` is not `NEEDS_INTERVIEW_TO_DETERMINE_LEVEL`